metals/src/main/scala/scala/meta/internal/metals/Configs.scala

```diff
@@ -72,6 +72,10 @@ object Configs {
           "metals.hover.documentation",
           default = true
         ),
+        isMagicIndentClient = MetalsServerConfig.binaryOption(
+          "metals.magic-indent-client",
+          default = false
```

Comment:
I think it would be better to default use current behavior, which is `true`. I have a feeling a lot of people on vim/emacs don't use the `client` property so that would be a regression for them.

Reply:
Sounds good. I changed the default to true